packages/api/src/routes/medical/internal-facility.ts

```diff
@@ -71,18 +70,17 @@ router.put(
     const syncInHie = await verifyCxItVendorAccess(cxId, false);
     // TODO Move to external/hie https://github.com/metriport/metriport-internal/issues/1940
     // CAREQUALITY
-    if (syncInHie && facility.cqApproved) {
+    if (syncInHie && facility.cqApproved && !skipHie) {
```

Comment:
We should prob combine both `syncInHie` and `skipHie` in a single one, prob `syncInHie` being initialized considering the value of `skipHie`?